api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -2969,4 +2969,101 @@ public void saveOrder_shouldPassIfAnKnownDrugOrderForTheSameDrugFormulationSpeci
 		orderService.saveOrder(order, orderContext);
 		assertNotNull(orderService.getOrder(order.getOrderId()));
 	}
+	
+	/**
+	 * @see OrderService#getNonCodedDrugConcept()
+	 * @verifies return an null if nothing is configured
+	 */
+	@Test
+	public void getNonCodedDrugConcept_shouldReturnNullIfNothingIsConfigured() throws Exception {
+		adminService.saveGlobalProperty(new GlobalProperty(OpenmrsConstants.GP_DRUG_NON_CODED_CONCEPT_UUID, ""));
+		assertNull(orderService.getNonCodedDrugConcept());
+	}
+	
+	/**
+	 * @see OrderService#getNonCodedDrugConcept()
+	 * @verifies return a Concept if GP is set
+	 */
+	@Test
+	public void getNonCodedDrugConcept_shouldReturnAConceptIfGPIsSet() throws Exception {
+		Concept nonCodedDrugConcept = orderService.getNonCodedDrugConcept();
+		assertNotNull(nonCodedDrugConcept);
+		assertThat(nonCodedDrugConcept.getConceptId(), is(5584));
+		assertEquals(nonCodedDrugConcept.getName().getName(), "DRUG OTHER");
+		
+	}
+	
+	/**
+	 * @verifies pass if an active drug order for the same coded concept, care setting and different drug non coded exists
+	 * @see OrderService#saveOrder(Order, OrderContext)
+	 */
+	@Test
+	public void saveOrder_shouldPassForANewNonCodedDrugOrderIfAnotherActiveNonCodedDrugOrderAlreadyExists()
```

Comment:
I think these tests are unnecessary because you already tested the same code in the tests for isSameOrderable